docs/download_files.md

```diff
@@ -33,28 +36,28 @@ The folder structure within the zip file is determined by whether individual sam
 Note that if a sample selected for download contains a spatial transcriptomics library, the files included will be different than pictured below.
 See the [description of the Spatial transcriptomics output section below](#spatial-transcriptomics-libraries).
 
-## `SingleCellExperiment` downloads
+### `SingleCellExperiment` downloads
 
 ### Download folder structure for project downloads:
 ![project download folder](images/project-download-folder.png)
 
 ### Download folder structure for individual sample downloads:
 ![sample download folder](images/sample-download-folder.png)
 
-## `AnnData` downloads
+### `AnnData` downloads
 
-### Download folder structure for project downloads:
+#### Download folder structure for project downloads:
 ![project download folder](images/anndata-project-download-folder.png)
 
-### Download folder structure for individual sample downloads:
+#### Download folder structure for individual sample downloads:
 ![sample download folder](images/anndata-sample-download-folder.png)
 
-### Download folder structure for individual sample downloads with CITE-seq (ADT) data:
+#### Download folder structure for individual sample downloads with CITE-seq (ADT) data:
 ![sample download folder](images/anndata-sample-citeseq-download-folder.png)
 
 If downloading a sample that contains a CITE-seq library as an `AnnData` object (`hdf5` file), the quantified CITE-seq expression data is included as a separate file.
 
```

Comment:
I think I get where you are going here, but I would move the merged object section to be immediately after all the download descriptions. It should be on the same level as `Gene expression data`, `QC report` etc. All of that is the same for both types of downloads. The only additional thing is the merge report which you can address in the text describing downloading projects as merged objects.

Reply:
Yea I see what you are saying. I think I would then leave everything at the level they were originally add, but then treat the merged object download as a special consideration section similar to spatial and multiplexed. So it should be on the same level as those items. Maybe after the multiplexed section and before the spatial section?

Reply:
> So it should be on the same level as those items. Maybe after the multiplexed section and before the spatial section?

Yeah, I'm good with this!